packages/server/modules/workspaces/tests/helpers/creation.ts

```diff
@@ -84,9 +102,19 @@ export type BasicTestWorkspace = {
 export const createTestWorkspace = async (
   workspace: SetOptional<BasicTestWorkspace, 'slug'>,
   owner: BasicTestUser,
-  options?: { domain?: string; addPlan?: boolean }
+  options?: { domain?: string; addPlan?: boolean; regionKey?: string }
 ) => {
-  const { domain, addPlan = true } = options || {}
+  const { domain, addPlan = true, regionKey } = options || {}
+  const useRegion = isMultiRegionTestMode() && regionKey
+
+  if (!FF_WORKSPACES_MODULE_ENABLED) {
+    // Just skip creation and set id to undefined - this allows this to be invoked the same way if FFs are on or off
```

Comment:
i can see how this is kinda hacky, but this allows me to write the same test code that works in both no-workspaces/with workspaces but not multiregion/with workspaces and multiregion mode

if there's better options, i'm open to it